sbol_utilities/sbol3_sbol2_conversion.py

```diff
@@ -546,9 +556,19 @@ def visit_measure(self, a: sbol2.measurement.Measurement):
         # Priority: 3
         raise NotImplementedError('Conversion of Measure from SBOL2 to SBOL3 not yet implemented')
 
-    def visit_model(self, a: sbol2.model.Model):
+    def visit_model(self, model2: sbol2.model.Model):
         # Priority: 3
-        raise NotImplementedError('Conversion of Model from SBOL2 to SBOL3 not yet implemented')
+        
+        model3 = sbol3.Model(model2.identity, source=model2.source, language=model2.language,   
+                             framework=model2.framework)
+        self.doc3.add(model3)
+        
+        # Map over all other TopLevel properties and extensions not covered by the constructor
+        self._convert_toplevel(model2, model3)
+
```

Comment:
Too much whitespace here; please use pycodestyle to follow standard python conventions..